### ISSUE TITLE
Please oh please let me hide #shorts in subs

### ISSUE THREAD

**Issue by pies** [OsaSoft/youtube-better-subscriptions#100]
Shorts are the devil.

**Comment by OsaSoft** [comment]
related (but not duplicate) to #69 (nice)

**Comment by ewonais** [comment]
can be done using advansesd blocking in BlockTube0
block it for specifik channels or for all

```
(video, objectType) => { 
  if (video.vidLength<=60) {     //if short
      if (video.channelName.match("channelA|channelB")) {return true; }
  }
  else {
      if (video.channelName.match("channelC")) {return true; } //reverse block shorts
  }
  return false;
}
```

**Comment by OsaSoft** [comment]
Ok! Version 0.17.0 which includes this, the fix in #105 and a few other fixes is uploaded to Mozilla and Chrome and should be going live as soon as they are approved!
